check/checker/types.go

```diff
@@ -16,6 +16,12 @@ package checker
 
 import v1 "github.com/alibaba/sealer/types/api/v1"
 
+// Checker: checking cluster status,such as node,pod,svc status.
 type Checker interface {
 	Check(cluster *v1.Cluster) error
 }
+
+// PreChecker: checking prerequisite for sealer,such as system info,register login status.
+type PreChecker interface {
```

Comment:
Why don't using the Checker interface? Some prechecker needs cluster.

Reply:
i think prechecker is a static checker ,such as systeminfo . but for cluster its a running state parameter。

Reply:
Some checker can ignore this argument. All the checkers implement the same interface